framework/src/systems/NonlinearSystemBase.C

```diff
@@ -851,6 +852,75 @@ NonlinearSystemBase::setConstraintSlaveValues(NumericVector<Number> & solution,
     }
   }
 
+  // go over NodeELemConstraints
+  bool nec_shouldApply = false;
+  std::set<dof_id_type> unique_slave_node_ids;
+  std::unique_ptr<PointLocatorBase> pointLocator = _fe_problem.mesh().getPointLocator();
```

Comment:
@permcody When you get a chance, we'd like to talk with you about how to do this with ReplicatedMesh.

Reply:
You probably mean DistributedMesh.

Reply:
Yeah, sorry -- that's what I meant.  Let me know when you've got a few minutes to talk about this.

Reply:
Right after lunch would work fine (~around noon or shortly thereafter).